scss/grid.scss

```diff
@@ -0,0 +1,95 @@
+.flex-body { 
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  grid-template-areas: 
+    "header"
+    "main"
+    "footer";
+
+  .train-header {
+    align-items: center;
+    grid-area: header;
+    top: 0;
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    grid-template-rows: auto;
+    grid-template-areas: 
+      "logo . dev";
```

Comment:
Same, delete template columns and rows if you use areas

Reply:
My bad here, if you are gonna specify column width (like the one that has 2fr) you DO need to add `grid-template-columns`.
Tho, you shoud use flexbod here instead and just use `space-between`